src/lib/validation.ts

```diff
@@ -377,11 +377,11 @@ export function isValidAppsObject(obj: any): boolean {
 		return false;
 	}
 
-	return _.every(obj, (v, appId) => {
-		if (!isValidShortText(appId) || !checkInt(appId)) {
+	return _.every(obj, (v, uuid) => {
+		if (!isValidShortText(uuid)) {
 			log.debug(
-				'Invalid appId passed to validation.isValidAppsObject\nApp ID:',
-				inspect(appId),
+				'Invalid uuid passed to validation.isValidAppsObject\nUUID:',
```

Comment:
```suggestion
				'Invalid UUID passed to validation.isValidAppsObject\nUUID:',
```

Reply:
Makes sense in code to follow naming conventions but in log messages UUID can be uppercase as acronyms usually are I believe.